addons/hr/static/src/views/fields/many2one_avatar_employee_field/many2one_avatar_employee_field.js

```diff
@@ -11,7 +11,14 @@ import { EmployeeFieldRelationMixin } from "@hr/views/fields/employee_field_rela
 
 export class Many2OneAvatarEmployeeField extends EmployeeFieldRelationMixin(
     Many2OneAvatarUserField
-) {}
+) {
+    get many2OneProps() {
```

Comment:
why not in the EmployeeFieldRelationMixin?

The commit message should no longer mention web

Reply:
> why not in the EmployeeFieldRelationMixin?

Because the mixin is also used for the KanbanMany2ManyTagsAvatarEmployeeField and Many2ManyTagsAvatarEmployeeField. Those don't contain a many2one field.

Reply:
I guess we could introduce another mixin, EmployeeFieldMany2OneMixin, but it's probably not worth it